web/config/app.go

```diff
@@ -0,0 +1,98 @@
+package config
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/beatlabs/github-auth/app"
+	"github.com/beatlabs/github-auth/key"
+	gh "github.com/google/go-github/v35/github"
```

Comment:
Should we upgrade to `github.com/google/go-github/v43/github`?? Not sure it is necessary, and what additional features are supported... didn't check. Perhaps some additional github apps features are supported...

Reply:
Upgraded to v43.